cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -274,12 +274,18 @@ function start-kube-apiserver {
   if [[ -n "${MASTER_ADVERTISE_ADDRESS:-}" ]]; then
     params+=" --advertise-address=${MASTER_ADVERTISE_ADDRESS}"
     if [[ -n "${PROXY_SSH_USER:-}" ]]; then
+      if [[ -n "${KUBE_API_SERVER_RUNASUSER:-}" && -n "${KUBE_API_SERVER_RUNASGROUP:-}" ]]; then
+        chown "${KUBE_API_SERVER_RUNASUSER}":"${KUBE_API_SERVER_RUNASGROUP}" /mnt/disks/master-pd/srv/sshproxy
```

Comment:
nit: shall we chown the symlink `/etc/srv/sshproxy` in case it changes in the future?

Reply:
Done. Thanks!